tools/ci/travis-install-dagmc.sh

```diff
@@ -20,7 +20,7 @@ mkdir MOAB && cd MOAB
 git clone -b $MOAB_BRANCH $MOAB_REPO
 mkdir build && cd build
 cmake ../moab -DENABLE_HDF5=ON -DBUILD_SHARED_LIBS=ON -DCMAKE_INSTALL_PREFIX=$MOAB_INSTALL_DIR
-make -j && make -j test install
+make -j2 && make -j install
```

Comment:
It looks like the memory issues resolved themselves, so I think it's ok to revert to `make -j`
```suggestion
make -j && make -j install
```